sdk/core/System.ClientModel/tests/Message/MultipartContentTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using ClientModel.Tests;
+using ClientModel.Tests.Mocks;
+using NUnit.Framework;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Mime;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Tests.Message;
+
+internal class MultipartContentTests : SyncAsyncTestBase
```

Comment:
Are there other tests we might want to consider adding? They may have relevant cases not covered here in https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/tests/FunctionalTests/MultipartContentTest.cs?

Reply:
Yes! I actually have a few additional tests locally that I'm working on, and I'll take a look at the ones you linked as well